redis/src/cluster_client.rs

```diff
@@ -229,6 +276,28 @@ impl ClusterClientBuilder {
         self
     }
 
+    /// Sets raw TLS certificates for the new ClusterClient.
+    ///
+    /// `tls` parameter is required since certificates are only used in TLS context
+    /// All certificates are byte streams loaded from PEM files
+    /// their consistency is checked during `build()` call
+    #[cfg(feature = "tls-rustls")]
+    pub fn certs(
+        mut self,
+        tls: TlsMode,
```

Comment:
Does `tls` need to be supplied with this method? Does it make sense to supply configuration for mutual TLS but to have `TlsMode` be insecure?